logging_json/json_log.py

```diff
@@ -13,17 +13,17 @@
 _logger = logging.getLogger(__name__)
 
 try:
-    from pythonjsonlogger import jsonlogger
+    from pythonjsonlogger.json import JsonFormatter
 except ImportError:
-    jsonlogger = None  # noqa
-    _logger.debug("Cannot 'import pythonjsonlogger'.")
+    JsonFormatter = None  # noqa
+    _logger.debug("Cannot 'import JsonFormatter from '.")
```

Comment:
Keep something easier to understand.
```suggestion
    _logger.debug("Cannot 'import pythonjsonlogger'.")
```